src/rust/include/rust/orchard/wallet.h

```diff
@@ -333,6 +333,18 @@ void orchard_wallet_get_potential_spends(
         push_txid_callback_t push_cb
         );
 
+/**
+ * Returns a vector of transaction IDs for transactions that have been observed as
+ * spending the given nullifier by using the `push_cb` callback to push transaction
+ * IDs onto the provided result vector.
+ */
+void orchard_wallet_get_potential_spends_from_nullifier(
+        const OrchardWalletPtr* wallet,
+        const uint256& nullifier,
```

Comment:
We don't want to depend on the C++ `uint256` type in the C FFI. Change this to:
```suggestion
        const unsigned char *nullifier,
```